content/_getting-started/first-flow.md

```diff
@@ -1,172 +1,123 @@
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_16.png)
 
-*   **Body** field: expects email's body as a `string`. Here is another JSONata expression to copy and paste:
-``
-"Darling, I found some adorable pets for us. Here are their names: " & $join(pets.name, ", ") &". Which one shall I buy?"
-``
+Optionally, set a name for the step. We're going to use Email Component for our third step. Select it and click "Choose Email":
 
-Click *Continue* after filling-in the form to continue.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_17.png)
 
-![Retrieve a sample from Email](/assets/img/getting-started/first-flow/getting-started-flow-012.png "Retrieve a sample from Email")
+Email Component sends an email when triggered, so you should just fill in the details as you would for an email. Also, you can cancel error message on the event of sending fail. After you've entered all the data, click "Continue":
 
-On the next page click on *Retrieve sample from E-Mail* to retrieve the sample.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_18.png)
 
-![Continue with the Email components' sample](/assets/img/getting-started/first-flow/getting-started-flow-013.png "Continue with the Email components' sample")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_19.png)
 
-When the sample gets retrieved click on *Choose this sample* to continue.
+Next, you retrieve a data sample and click "Continue":
 
-![Almost done with the flow](/assets/img/getting-started/first-flow/getting-started-flow-014.png "Almost done with the flow")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_20.png)
 
-We have finished configuring this demonstration flow. You could add more steps
-after this moment but we recommend you not to at this stage. Click on *I’m done*
-button once.
+Finally, you may add a description and click "Finish step":
 
-![Save and continue](/assets/img/getting-started/first-flow/getting-started-flow-015.png "Save and continue")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_21.png)
 
-You can name your integration flow or accept the suggested name by clicking on
-*I'm done* one more time to finish and save the current state of your Draft.
+To publish the Flow draft and be able to run the Flow, click the corresponding button:
 
-> **Note**: at this stage you can still modify this draft flow.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_22.png)
 
-![Publish the draft](/assets/img/getting-started/first-flow/getting-started-flow-016.png "Publish the draft")
 
-Before the flow can be used we need to publish the current draft. After you
-publish this flow you cannot change this version of the flow, unless you create a
-[new draft of this flow](/integrator-guide/managing-flow-history). Click on *Publish Draft*
-button to finish modifying it and saving this version of the flow.
 
-## Running and monitoring the flow
+## Post-Creation Activities
```

Comment:
I think we should move this into separate articles. The topics covered here are too heavy for beginners. Maybe we should just tell the user how to start the flow, how go to Dashboard and see the execution results. What do you think?